server/service/src/invoice/inbound_shipment/update/generate.rs

```diff
@@ -67,6 +83,44 @@ pub fn should_create_batches(invoice: &InvoiceRow, patch: &UpdateInboundShipment
     }
 }
 
+// If status changed to Picked and above, remove unallocated lines
```

Comment:
Should it be `// If status changed to Verified and above, remove unallocated lines` ?